src/System/Drawing/Document.ts

```diff
@@ -414,47 +475,28 @@ export default class Document extends Renderable
                 }
             };
         }
-
-        // Preparing the attributes
-        let view = {};
-
-        for (let key in this.Attributes)
-        {
-            let value = this.Attributes[key];
-
-            if (value instanceof Date || Date.parse(value))
-            {
-                value = new DateTimeFormatter(this.Locale).Format(this.DateFormat, new Date(value));
-            }
-
-            view[key] = value;
-        }
-
-        let html = md.render(content);
-        return Mustache.render(html, view);
+        return md;
     }
 
     /**
      * Renders the body of the document.
      */
     public async Render(): Promise<string>
     {
-        let styleCode = "<style>\n";
+        let styleCode = "";
+        let scriptCode = "";
 
         for (let styleSheet of this.StyleSheets)
         {
             if (/.*:\/\//g.test(styleSheet) || !Path.isAbsolute(styleSheet))
             {
-                styleCode += Dedent(`
-                    </style>
-                    <link rel="stylesheet" type="text/css" href="/${styleSheet}" />
-                    <style>`);
+                styleCode += Dedent(`<link rel="stylesheet" type="text/css" href="/${styleSheet}" />\n`);
```

Comment:
Oh yeah, you could, btw, safely remove the `Dedent`-wrapper.
It's a great tool for removing indentation from your strings while still keeping your code clean.